project/app/api/reports.py

```diff
@@ -66,5 +67,11 @@ async def update_report(
 # must be superuser
 # delete a report
 @router.delete("/", response_model=Report_Pydnatic)
-async def delete_report(report_id: int):
-    await Report.filter(id=user_id).delete()
+async def delete_report(
+        report_id: int,
+        current_superuser: User = Depends(find_current_superuser)
+):
+    deleted_report = await Report.filter(id=report_id).delete()
+    if not deleted_report:
+        raise HTTPException(status_code=404, detail=f"Report {report_id} not found")
+    return Status(message=f"Delete Report {report_id}")
```

Comment:
I don't see a Status class anywhere. this should return a JSONResponse instead. Look at #13 comments instead.